modular_skyrat/modules/xenoarch/code/modules/research/xenoarch/xenoarch_reward.dm

```diff
@@ -0,0 +1,76 @@
+GLOBAL_LIST_INIT(tier1_reward, list(
+	/obj/item/xenoarch/useless_relic = 5,
+	/obj/item/candle = 1,
+	/obj/item/trash/chips = 1,
+	/obj/item/trash/raisins = 1,
+	/obj/item/trash/cheesie = 1,
+	/obj/item/trash/candy = 1,
+))
+
+GLOBAL_LIST_INIT(tier2_reward, list(
+	/obj/item/xenoarch/broken_item/tech = 1,
+	/obj/item/xenoarch/broken_item/plant = 1,
+	/obj/item/xenoarch/useless_relic = 5,
+))
+
+GLOBAL_LIST_INIT(tier3_reward, list(
+	/obj/item/xenoarch/broken_item/weapon = 3,
+	/obj/item/xenoarch/broken_item/illegal = 1,
+	/obj/item/xenoarch/broken_item/alien = 1,
+	/obj/item/stack/spacecash/c10000 = 1,
```

Comment:
I'm not sure just casually dropping 10000 credits is a good idea.

Reply:
1000 would probably be fine, or even 500.